zod/src/__tests__/__snapshots__/zod.ts.snap

```diff
@@ -375,3 +401,30 @@ Object {
   "values": Object {},
 }
 `;
+
+exports[`zodResolver should return parsed values from zodResolver with \`mode: sync\` when validation pass 1`] = `
+Object {
+  "errors": Object {},
+  "values": Object {
+    "accessToken": "accessToken",
+    "birthYear": 2000,
+    "dateStr": 2020-01-01T00:00:00.000Z,
```

Comment:
You see here @bluebill1049, the default behavior is to apply the transforms which will give a `Date` here and not a string